<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TileSetDatabaseWidget</class>
 <widget class="QWidget" name="TileSetDatabaseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TileSetDatabaseWidget</string>
  </property>
  <widget class="QTableWidget" name="m_pTileSetTable">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>160</width>
     <height>160</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::DropOnly</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>0</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>32</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>32</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>32</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>32</number>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TileSet:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pHorizontalAdd">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pHorizontalRemove">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pVerticalAdd">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pVerticalRemove">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_pSpriteLabel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>0</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>tiles:</string>
   </property>
  </widget>
  <widget class="TilePrototypeDragTreeWidget" name="m_pTileList">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>231</width>
     <height>271</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="indentation">
    <number>0</number>
   </property>
   <property name="rootIsDecorated">
    <bool>false</bool>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="headerHidden">
    <bool>false</bool>
   </property>
   <property name="expandsOnDoubleClick">
    <bool>false</bool>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="headerVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="headerCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="headerDefaultSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="headerHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="headerShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TilePrototypeDragTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>dragtreewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
